### ISSUE TITLE
Make a STABLE release for Customer., like how windows have the stable installer for installing it like that VINIX should have.

### ISSUE THREAD

**Issue by athrvvvv** [vlang/vinix#81]
If the software will be realeased than it could cover more and more people. So Make a STABLE release for Customer

**Comment by MatusModder** [comment]
No. I think that Vinix should be like Arch or Gentoo (rolling release). They do not have installers. And btw, Vinix is open-source project. I do not think that autor of this OS should make money of off it. If you want to donate, you can donate money via GitHub Sponsors.
